src/Components/Web.JS/src/webpack.config.js

```diff
@@ -11,5 +11,8 @@ module.exports = (env, args) => ({
         'blazor.webassembly': './Boot.WebAssembly.ts',
         'blazor.server': './Boot.Server.ts',
     },
-    output: { path: path.join(__dirname, '/..', '/dist', args.mode == 'development' ? '/Debug' : '/Release'), filename: '[name].js' }
+    output: { path: path.join(__dirname, '/..', '/dist', args.mode == 'development' ? '/Debug' : '/Release'), filename: '[name].js' },
+    performance: {
```

Comment:
Hm -- why was this added?

Reply:
@BrennanConroy Can this get the same treatment as #30035 so we're not updating the TypeScript compiler and a bunch of extraneous files in a release branch?

Reply:
> Hm -- why was this added?

Webpack warns about assets being larger than 244Kib in size. The blazor.server.js file was already right up against that limit so any modifications could cause the warning.

Reply:
Changed to `maxAssetSize` so we still get warnings if the size increases again.

Reply:
> Can this get the same treatment as #30035 so we're not updating the TypeScript compiler and a bunch of extraneous files in a release branch?

Done

Reply:
`maxAssetSize` seems useful. Should we add `hints: "error"` to this and fail the build if the limit is exceeded?

If an increase in size is deemed worth it down the road we can always change the limit. I don't know if 270KiB leaves enough wiggle room for that to be a good idea considering we're already up against the default 244KiB limit.

This is all more important in the `main` branch ofc. I'm not sure we should change this at all in a release branch unless it's already breaking the build.

Reply:
I think it warns by default, and our build fails on warnings in yarn builds.

I think the limit is fine, because if the size does increase again we'd like to know about it. Even if we just end up upping it again.